jedi/evaluate/sys_path.py

```diff
@@ -30,11 +31,19 @@ def check_virtual_env(sys_path):
     return [p for p in sys.path if p != ""]
 
 def _get_venv_sitepackages(venv):
+    pyre = re.compile('^python\d\.\d$')
```

Comment:
Fair enough. I'd originally avoided an re for speed but since this should be memoized there shouldn't be much of a hit.